reliability-engineering/terraform/modules/concourse-secrets-admin/iam.tf

```diff
@@ -0,0 +1,82 @@
+resource "aws_iam_role" "concourse_secrets_admin" {
+  name = "${var.deployment}-${var.concourse_team_name}-concourse-secrets-admin"
+
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Effect = "Allow"
+        Principal = {
+          Federated = module.concourse_base.aws_iam_openid_connect_provider_arn
```

Comment:
how can we refer to module.concourse_base here?

Reply:
Replaced with input variable.